children/tests/test_api.py

```diff
@@ -821,6 +887,57 @@ def test_child_past_enrolment_count(
     assert executed["data"]["child"]["pastEnrolmentCount"] == in_the_past
 
 
+@freeze_time("2021-02-02T12:00:00Z")
+@pytest.mark.parametrize("in_the_past", [0, 1, 2])
+def test_child_past_enrolment_count_with_ticket_system_passwords(
+    guardian_api_client,
+    child_with_user_guardian,
+    in_the_past,
```

Comment:
This could be renamed to e.g. `past_enrolment_count` to make it more explicit what the variable means (Zen of Python → "Explicit is better than implicit.").

Reply:
Taken into account.